pysrc/bytewax/outputs.py

```diff
@@ -34,29 +34,46 @@ def TestingEpochOutputConfig(ls):
 
     Returns:
 
-        Config object. Pass this to the
-        `bytewax.dataflow.Dataflow.capture` operator.
+        Config object.
+        Pass this to the `bytewax.dataflow.Dataflow.capture` operator.
 
     """
-    return ManualEpochOutputConfig(lambda wi, wn: ls.append)
+    # This is needed to avoid pytest trying to load this class as
+    # a test since its name starts with "Test"
+    __test__ = False
 
+    def __new__(cls, ls):
+        """
+        In classes defined by PyO3 we can only use __new__, not __init__
```

Comment:
Small nit: feels like this comment should be above the function, rather than documentation of it as it's not integral to its functionality as much as to its implementation.